src/plone/restapi/tests/test_services_mail.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
```

Comment:
@sneridagh the name of the test file should be in sync with the file where the actual code is. -> test_services_email_notification.

Reply:
Done.